bcda/auth/client/oktaclient_test.go

```diff
@@ -74,6 +74,22 @@ func (s *OTestSuite) TestAddClientApplication() {
 	assert.NotEmpty(s.T(), secret)
 }
 
+func (s *OTestSuite) TestRequestAccessToken() {
+	clientID := os.Getenv("OKTA_CLIENT_ID")
+	clientSecret := os.Getenv("OKTA_CLIENT_SECRET")
+
+	assert.NotEmpty(s.T(), clientID, "Test requires OKTA_CLIENT_ID")
+	assert.NotEmpty(s.T(), clientSecret, "Test requires OKTA_CLIENT_SECRET")
+
+	t, err := s.oc.RequestAccessToken(Credentials{ClientID: clientID, ClientSecret: clientSecret})
+	assert.IsType(s.T(), OktaToken{}, t)
+	assert.Nil(s.T(), err)
+
+	t, err = s.oc.RequestAccessToken(Credentials{ClientID: "", ClientSecret: ""})
+	assert.IsType(s.T(), OktaToken{}, t)
+	assert.NotNil(s.T(), err)
+}
+
 func (s *OTestSuite) TestGenerateNewClientSecret() {
 	validClientID := "0oaj4590j9B5uh8rC0h7"
```

Comment:
we can use your os.Getenv("OKTA_CLIENT_ID") here if you want to make this change while you're at it

Reply:
But then it would change the secret I carefully put in my environment.  😄

Reply:
this is just the clientID.  not the secret

Reply:
But the following code changes the secret associated with it.  We'd have to have a separate client ID for this test.  I'll back down from my previous position.